codex-ui/src/vue/components/popup/Confirm.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div
+    :class="$style['confirm']"
+    data-dimensions="large"
+  >
+    <div :class="['text-ui-base-bold']">
+      {{ title }}
+    </div>
+    <div :class="[$style['confirm__body'], 'text-ui-base-medium']">
+      {{ body }}
+    </div>
+    <div :class="$style['confirm__controls']">
+      <Button
+        secondary
+        @click="onCancel"
+      >
+        <div :class="$style['confirm__button-inner']">
+          Cancel
```

Comment:
don't we need i18n?

Reply:
I think we'll add i18n when we implement the component in the app - there's no need for localization right now, as the component is only added to Playground